app/configurator/interactive-filters/time-slider.tsx

```diff
@@ -226,7 +140,7 @@ const PlayButton = () => {
         borderRadius: "50%",
       }}
     >
-      <Icon name={animating ? "pause" : "play"} size={16} />
+      <Icon name={timeline?.playing ? "pause" : "play"} size={16} />
```

Comment:
To prevent this optional operator, I suggest that useTimeline throws if not inside the context, this way we can reflect in the type that the result of useTimeline is Timeline and not Timeline | undefined.